openedx/adg/lms/applications/models.py

```diff
@@ -66,7 +66,7 @@ def percentage_of_objectives_completed(self):
         """
         number_of_objectives_completed = sum([self.is_written_application_completed,
                                               self.is_prerequisite_courses_passed])
-        return '{0:.0%}'.format(number_of_objectives_completed / self.TOTAL_APPLICATION_OBJECTIVES)
+        return number_of_objectives_completed / self.TOTAL_APPLICATION_OBJECTIVES
```

Comment:
We should multiply it with 100. That would align with the name of the method. But it is not just to align with the name of the method.